pynicotine/gtkgui/search.py

```diff
@@ -1418,30 +1411,51 @@ def on_refilter(self, *_args):
         if self.clearing_filters:
             return
 
-        filter_in = self.filter_include_combobox.get_active_text().strip().lower()
-        filter_out = self.filter_exclude_combobox.get_active_text().strip().lower()
+        filter_in = self.filter_include_combobox.get_active_text().strip()
+        filter_out = self.filter_exclude_combobox.get_active_text().strip()
+        filter_size = self.filter_file_size_combobox.get_active_text().strip()
+        filter_bitrate = self.filter_bitrate_combobox.get_active_text().strip()
+        filter_country = self.filter_country_combobox.get_active_text().strip()
+        filter_file_type = self.filter_file_type_combobox.get_active_text().strip()
+        filter_length = self.filter_length_combobox.get_active_text().strip()
+        filter_free_slot = self.filter_free_slot_button.get_active()
 
         if filter_in:
             try:
-                filter_in = re.compile(filter_in)
+                filter_in = re.compile(filter_in.lower())
             except sre_constants.error:
                 filter_in = None
 
         if filter_out:
             try:
-                filter_out = re.compile(filter_out)
+                filter_out = re.compile(filter_out.lower())
             except sre_constants.error:
                 filter_out = None
 
+        if filter_size:
+            filter_size = filter_size.replace(" ", "").split("|")
```

Comment:
These `replace(" ", "")` operations make the `strip()` operations above redundant.

Reply:
Problem is, if you just have whitespace as the value, say " ", you end up with an empty string in the list after splitting [""].

Reply:
Reduced it to a single operation in bceb85700c4044cce604bd11947a6bb67c76e1ee